src/Core/System.php

```diff
@@ -549,7 +548,7 @@ private static function isDirectoryUsable($directory, $check_writable = true)
 			return false;
 		}
 
-		if ($check_writable && !is_writable($directory)) {
+		if (!is_writable($directory)) {
```

Comment:
What about the places that used the second parameter that you removed?

Reply:
There is no usage of the second parameter, my IDE suggested removing it because of that :-)